mindbot.py

```diff
@@ -35,12 +36,201 @@ async def checkUserId(interaction: discord.Interaction, usertag: str):
+	override = overrideFieldAsDict["value"]
+			
+	return name, power, capabilities, effect, quote, cardnumber, setname, language, filename, str2bool(override)
+#endregion
+
+#region UI STUFF
+class EditCardData(ui.Modal, title='Edit Card Name'):
+	nameInput = ui.TextInput(label='Name', placeholder="Sirus Snape", required=True)
+	powerInput = ui.TextInput(label='Power', placeholder="9", required=True)
+	capabilitiesInput = ui.TextInput(label='Keywords', placeholder="Frenzy, Tough", required=False)
+	effectInput = ui.TextInput(label='Effect', placeholder="#Play: Draw a card. #Defeat: Gain +1 HP.", required=False)
+	quoteInput = ui.TextInput(label='Quote', placeholder="Nothing special", required=False)
+	
+	async def on_submit(self, interaction: discord.Interaction):
+
+		await interaction.response.defer(ephemeral=True, thinking=True)
+		# Update the Mindbug-Card
+		name, power, capabilities, effect, quote, cardnumber, setname, language, filename, override= getCardDataFromEmbed(interaction)
+
+		myEmbed = discord.Embed(color = discord.Color.random())
+		myEmbed.add_field(name="Name", value=self.nameInput.value.strip(), inline=True) # Index 0
+		myEmbed.add_field(name="Power", value=self.powerInput.value.strip(), inline=False) # Index 1
+		myEmbed.add_field(name="Capabilities", value=self.capabilitiesInput.value.strip() if (len(self.capabilitiesInput.value)  != 0 ) else "?", inline=False) # Index 2
+		myEmbed.add_field(name="Effect", value=self.effectInput.value.strip() if (len(self.effectInput.value)  != 0 ) else "?"  , inline=False) # Index 3
+		myEmbed.add_field(name="Quote", value=self.quoteInput.value.strip() if (len(self.quoteInput.value) != 0 ) else "?" , inline=False) # Index 4
+		myEmbed.add_field(name="Cardnumber in Set", value=cardnumber, inline=False) # Index 5
+		myEmbed.add_field(name="Name from Set", value=setname, inline=False) # Index 6
+		myEmbed.add_field(name="Language", value=language, inline=False) # Index 7
+		myEmbed.add_field(name="Filename", value=f"{filename}", inline=False) # Index 8
+		myEmbed.add_field(name="Override", value=f"{override}", inline=False) # Index 9
+	
+		await interaction.followup.send(embed = myEmbed, view = EditMenu(), ephemeral=True)
+
+class EditCardMetaData(ui.Modal, title='Edit Card Name'):
+	
+	languageInput = ui.TextInput(label='Language', placeholder="en", required=True)
+	setnameInput = ui.TextInput(label='Card Set', placeholder="Second Wave", required=True)
+	uid_from_setInput = ui.TextInput(label='Cardnumber', placeholder="1/24", required=True)
+	
+	async def on_submit(self, interaction: discord.Interaction):
+
+		await interaction.response.defer(ephemeral=True, thinking=True)
+		# Update the Mindbug-Card
+		name, power, capabilities, effect, quote, cardnumber, setname, language, filename, override = getCardDataFromEmbed(interaction)
+
+		myEmbed = discord.Embed(color = discord.Color.random())
+		myEmbed.add_field(name="Name", value=name, inline=True) # Index 0
```

Comment:
in the field description, would be good to have the allowed values (like true/false for override or "en" for default language)